ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/block/AbstractBlockProcessor.java

```diff
@@ -81,11 +81,16 @@
 import tech.pegasys.teku.spec.logic.versions.deneb.block.KzgCommitmentsProcessor;
 
 public abstract class AbstractBlockProcessor implements BlockProcessor {
+
+  @VisibleForTesting
+  public static final BLSSignatureVerifier DEFAULT_DEPOSIT_SIGNATURE_VERIFIER =
+      BLSSignatureVerifier.SIMPLE;
   /**
    * For debug/test purposes only enables/disables {@link DepositData} BLS signature verification
    * Setting to <code>false</code> significantly speeds up state initialization
    */
-  public static boolean blsVerifyDeposit = true;
+  @VisibleForTesting
+  public static BLSSignatureVerifier depositSignatureVerifier = DEFAULT_DEPOSIT_SIGNATURE_VERIFIER;
```

Comment:
are we planning on differentiating? why not just reference the default_deposit_signature_verifier?

Reply:
`DEFAULT_DEPOSIT_SIGNATURE_VERIFIER` was intended as a default value for resetting the `depositSignatureVerifier` field after testing with disabled verifier. But you are right, it's probably an over-complication for the production code.